enterprise/functions/src/main/java/io/crate/window/NthValueFunctions.java

```diff
@@ -38,9 +38,9 @@
 
 public class NthValueFunctions implements WindowFunction {
 
-    private static final String LAST_VALUE_NAME = "last_value";
-    private static final String FIRST_VALUE_NAME = "first_value";
-    private static final String NTH_VALUE = "nth_value";
+    public static final String LAST_VALUE_NAME = "last_value";
+    public static final String FIRST_VALUE_NAME = "first_value";
```

Comment:
is the First and Nth value used somewhere outside the class? do they need to be public?

Reply:
you are right, I changed all of them for consistency's sake and since they are static final. I can change back to being public.
What do we prefer in such cases? consistency or exposure as less as possible?

Reply:
I think we should only expose what's necessary

Reply:
fair enough, I'll change it back to private